README.md

```diff
@@ -59,7 +59,12 @@ In addition to the options accepted by the syslog (compliant with [RFC 3164][1]
 * __localhost:__ Host to indicate that log messages are coming from (Default: `localhost`).
 * __type:__ The type of the syslog protocol to use (Default: `BSD`, also valid: `'3164'`, `'5424'`, `'RFC3164'` or `'RFC5424'`).
 * __app_name:__ The name of the application (Default: `process.title`).
-* __eol:__ The end of line character to be added to the end of the message (Default: Message without modifications).
+* __eol:__ The end of line character to be added to the end of the message 
+(Default: Message without modifications).
+* __checkServerIdentity:__ check TLSScoket options https://nodejs.org/api/tls.html#class-tlstlssocket
+* __secureProtocol:__ check TLSScoket options https://nodejs.org/api/tls.html#class-tlstlssocket
+* __ciphers:__ check TLSScoket options https://nodejs.org/api/tls.html#class-tlstlssocket
+* __ecdhCurve:__ check TLSScoket options https://nodejs.org/api/tls.html#class-tlstlssocket
```

Comment:
Please check spelling on `TLSScoket`

Reply:
Thank you so much. I fixed the readme file. 